tic2.js

```diff
@@ -0,0 +1,107 @@
+const readline = require('readline');
+
+const readlineInterface = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        readlineInterface.question(questionText, resolve);
+    });
+}
+
+let cells = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+let divider = '--- --- ---'
+let player = { turn: 'X' }
+
+let printGrid = function () {
+    console.log(('\n ' + cells[0] + ' | ' + cells[1] + ' | ' + cells[2] + '\n'
+        + divider + '\n '
+        + cells[3] + ' | ' + cells[4] + ' | ' + cells[5] + '\n'
+        + divider + '\n '
+        + cells[6] + ' | ' + cells[7] + ' | ' + cells[8]))
+}
+
+let askForInput = async function () {
+    let input = await ask('\n> ');
```

Comment:
`input` will be a string; it's a good idea to immediately convert it into a number e.g. `input = +(input)` or `input = parseInt(input)`.